examples/scroll_colors.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This example allows to play with scroll bars over different color tones.
+
+use druid::shell::kurbo::{Rect, Size};
+use druid::shell::piet::{Color, RenderContext};
+use druid::shell::{runloop, WindowBuilder};
+use druid::widget::{Column, Row, Scroll};
+use druid::{
+    Action, BaseState, BoxConstraints, Data, Env, Event, EventCtx, LayoutCtx, PaintCtx, UiMain,
+    UiState, UpdateCtx, Widget,
+};
+
+struct FixedSize {
+    size: Size,
+    color: Color,
+}
+
+impl FixedSize {
+    pub fn new(size: Size, color: Color) -> Self {
+        Self { size, color }
+    }
+}
+
+impl<T: Data> Widget<T> for FixedSize {
```

Comment:
This is in an example, so less important, but I'm wondering if its use could be replaced with `SizedBox` now that's landed.

Reply:
There is a reason below:

>       // There is a SizedBox widget but it's not currently possible
>       // to set a background on it.

I wanted to use `SizedBox` but we currently have no way to style it. Would be nice to discuss how we do basic stylings (background, border, ...)